### PR TITLE
Registry improvements

### PR DESCRIPTION
**Pull request by randy-sab-roy**
## 📖 Description

Replaces the documentation for the container registry to use Gitea. Also adds a script to facilitate tagging new images without having to pull them first.

### CODE REVIEW

File: registry/README.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-Tart supports uploading images to an OCI registry. This section describes how to get a minimal registry up and running.
+Tart supports uploading images to an OCI registry. This section describes how to setup Gitea as a container registry.
 
 ## Prerequisite
 
-- Install [Docker](https://www.docker.com/)
-- Install `htpasswd` (preinstalled on MacOS)
+```
+$ brew install colima docker
+$ colima start
```
Comment (remi) [inline registry/README.md:9]: Should we have a `launchctl` service for the registry as well?

File: registry/docker-compose.yaml [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+    external: false
+
+volumes:
+  gitea:
+    driver: local
+
 services:
-  registry:
+  server:
+    image: gitea/gitea:1.18.5
```
Comment (simonlary) [inline registry/docker-compose.yaml:13]: We could at least use the latest 1.19. 😜

**Review by simonlary — APPROVED** [review]

File: registry/docker-compose.yaml (hunk 2 above)
Comment (randy-sab-roy) [reply registry/docker-compose.yaml:13]: 1.19 was not out when I did this. I'll update it!

File: registry/README.md (hunk 1 above)
Comment (randy-sab-roy) [reply registry/README.md:9]: Yes! (probably in another PR)

**Review by remi — APPROVED** [review]

**Pull request merged**